packages/mds-compliance/helpers.ts

```diff
@@ -20,7 +27,30 @@ export function clientCanViewPolicyCompliance(
           policy.provider_ids.length !== 0 &&
           queried_provider_id &&
           policy.provider_ids.includes(queried_provider_id)) ||
-          !policy.provider_ids ||
-          policy.provider_ids.length === 0)))
+          isPolicyUniversal(policy))))
   )
 }
+
+export async function feedInputsToComplianceEngine(
```

Comment:
I think that a bit of a clearer refactor would be to have this method be `getComplianceInputs`, and to remove the last line where we actually call out to compliance to process (and keep that line in the original place). If you really want to keep it structured this same way, I think that `feedInputsToComplianceEngine` is a bit of a misnomer, this is going and getting the necessary inputs and then asking the engine to process them -- not really feeding into the engine directly. Perhaps `getInputsAndComputeCompliance`?

Reply:
👍

Reply:
Out of scope for now, but I think there's a lot of stuff in the API layer that ought to be pushed into the engine at some point.

Reply:
Agreed 150%